presto-main/src/test/java/com/facebook/presto/operator/project/TestPageProcessor.java

```diff
@@ -335,7 +339,8 @@ public void testYieldProjection()
         Arrays.fill(slices, Slices.allocate(rows));
         Page inputPage = new Page(createSlicesBlock(slices));
 
-        PageProcessorOutput output = pageProcessor.process(SESSION, yieldSignal, inputPage);
+        AggregatedMemoryContext memoryContext = newSimpleAggregatedMemoryContext();
+        Iterator<Optional<Page>> output = pageProcessor.process(SESSION, yieldSignal, memoryContext, inputPage);
```

Comment:
maybe add an overload so we can use `processAndAssertRetainedPageSize` for all of these